frontend/package.json

```diff
@@ -195,7 +195,8 @@
     "react-dom": "^17.0.1",
     "react-draggable": "4.x",
     "react-helmet-async": "^2.0.5",
-    "react-i18next": "^11.7.3",
+    "react-helmet": "^6.1.0",
+    "react-i18next": "^11.18.6",
```

Comment:
I think this is what is causing the test failures:
`package.json (1 errors)
    Console provides shared module react-i18next ^11.18.6 but plugin uses version 11.12.0
[webpack-cli] Failed to load '/go/src/github.com/openshift/console/dynamic-demo-plugin/webpack.config.ts' config
[webpack-cli] Error: Validation failed`